lib/actions/ResourcesRemove.js

```diff
@@ -0,0 +1,215 @@
+'use strict';
+
+/**
+ * Action: ResourcesRemove
+ * - Removes the cloudformation/resources-cf.json template from AWS
+ */
+
+module.exports = function(SPlugin, serverlessPath) {
+
+  const path   = require('path'),
+    replaceall = require('replaceall'),
+    SError     = require(path.join(serverlessPath, 'ServerlessError')),
+    SCli       = require(path.join(serverlessPath, 'utils/cli')),
+    BbPromise  = require('bluebird'),
+    SUtils     = require(path.join(serverlessPath, 'utils/index')),
+    _          = require('lodash'),
+    fs         = BbPromise.promisifyAll(require('fs'));
+
+  class ResourcesRemove extends SPlugin {
+
+    /**
+     * Constructor
+     */
+
+    constructor(S, config) {
+      super(S, config);
+    }
+
+    /**
+     * Define your plugins name
+     */
+
+    static getName() {
+      return 'serverless.core.' + ResourcesRemove.name;
+    }
+
+    /**
+     * @returns {Promise} upon completion of all registrations
+     */
+
+    registerActions() {
+      this.S.addAction(this.resourcesRemove.bind(this), {
+        handler:       'resourcesRemove',
+        description:   `Remove AWS resources (resources-cf.json).
+usage: serverless resources remove`,
+        context:       'resources',
+        contextAction: 'remove',
+        options:       [
+          {
+            option:      'region',
+            shortcut:    'r',
+            description: 'region you want to remove from'
+          },
+          {
+            option:      'stage',
+            shortcut:    's',
+            description: 'stage you want to remove from'
+          },
+          {
+            option:      'noExeCf',
+            shortcut:    'c',
+            description: 'Optional - Don\'t execute CloudFormation, just remove it. Default: false'
+          }
+
+        ]
+      });
+      return BbPromise.resolve();
+    }
+
+    /**
+     * Action
+     */
+
+    resourcesRemove(evt) {
+
+      let _this    = this;
+      _this.evt    = evt;
+
+      return _this._prompt()
+        .bind(_this)
+        .then(_this._validateAndPrepare)
+        .then(_this._removeResources)
+        .then(function() {
+
+          /**
+           * Return EVT
+           */
+
+          return _this.evt;
```

Comment:
Can you pass on the name of the stack that was just deleted by attaching it to `evt.data`.  

Simply add `evt.data.stackName`